IntegrationTests/Tests/DocCConvertSynthesizedSymbolsTests.swift

```diff
@@ -0,0 +1,49 @@
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+
+import XCTest
+
+final class DocCConvertSynthesizedSymbolsTests: XCTestCase {
```

Comment:
```suggestion
final class DocCConvertSynthesizedSymbolsTests: ConcurrencyRequiringTestCase {
```

This should fix the test failures on macOS. We currently don't have a Toolchain that supports Swift concurrency on the macOS CI.